libs/blocks/merch/autoblock.js

```diff
@@ -35,9 +35,62 @@ export async function decorateCardCtasWithA11y(element, self) {
   });
 }
 
+export function handleCustomAnalyticsEvent(eventName, element) {
+  let daaLhValue = '';
+  let daaLhElement = element.closest('[daa-lh]');
+  while (daaLhElement) {
+    if (daaLhValue) {
+      daaLhValue = `|${daaLhValue}`;
+    }
+    const daaLhAttrValue = daaLhElement.getAttribute('daa-lh');
+    daaLhValue = `${daaLhAttrValue}${daaLhValue}`;
+    daaLhElement = daaLhElement.parentElement.closest('[daa-lh]');
+  }
+  if (daaLhValue) {
+    console.log('daaLhValue: ', `${eventName}|${daaLhValue}`);
+    // eslint-disable-next-line no-underscore-dangle
+    window._satellite?.track('event', {
+      xdm: {},
+      data: { web: { webInteraction: { name: `${eventName}|${daaLhValue}` } } },
+    });
+  }
+}
+
+export function getTabNestedLh(el) {
+  const tabPanel = el.closest('.tabpanel');
+  const tabDaaLh = tabPanel?.getAttribute('data-nested-lh');
```

Comment:
what is data-nested-lh?

Reply:
attribute that tabs block is setting on tabpanel, it identifies the tab, e.g. T1Ind. 
https://github.com/adobecom/milo/blob/stage/libs/blocks/tabs/tabs.js#L388